### PR TITLE
Remove Immunizations and Vaccinations

### PR DESCRIPTION
**Pull request by patfreeman**
Closes out #41

### CODE REVIEW

**Comment by ph1l** [comment]
Since this isn't related to the porting effort, resubmit as a PR to master, after I mere the porting branch to master... shortly

**Comment by patfreeman** [comment]
The restore would restore the data to a pre-migrated state, so it'd have all the tables and foreign columns in the parent tables. I'm not sure how the app would act if it were spun up on that state, but anytime a restore is done, a db migration should be run to bring the db in line with the applications expectations for the database.

**Comment by ph1l** [comment]
oh yea, fair enough..

File: util/initialize_mysql_db.sh.orig [hunk 1]
```diff
@@ -0,0 +1,82 @@
+#!/bin/bash -e
```
Comment (ph1l) [inline util/initialize_mysql_db.sh.orig]: This `.orig` file is likely a mistaken addition.

Comment (patfreeman) [reply util/initialize_mysql_db.sh.orig]: Do you want a clean branch, or should I just delete this in a second commit?

Comment (ph1l) [reply util/initialize_mysql_db.sh.orig]: I'd build a new commit/branch and force push. that keeps the mistake out of the history...

**Pull request merged**